addon/components/paper-item.js

```diff
@@ -82,10 +82,14 @@ export default Component.extend(RippleMixin, ProxyMixin, {
         let el = component.$();
         set(this, 'mouseActive', false);
         el.on('mousedown', ()=> {
-          set(this, 'mouseActive', true);
-          run.later(()=> {
-            set(this, 'mouseActive', false);
-          }, 100);
+          if (!get(component, 'isDestroyed') && !get(component, 'isDestroying')) {
```

Comment:
Why not? clicking on paper-item should create a ripple effect according to the md spec & AM impl?